workspace_tools/export/iar_mts_dragonfly_f411re.ewp.tmpl

```diff
@@ -627,7 +627,7 @@
       <archiveVersion>1</archiveVersion>
       <data>
         <prebuild></prebuild>
-        <postbuild></postbuild>
+        <postbuild>"$PROJ_DIR$\post-build.bat" "$TARGET_DIR$"</postbuild>
```

Comment:
Our process for exporting to IAR isn't completely automated because there doesn't seem to be a mechanism to insert additional files into the zip that gets created by the exporter.  We need to put in the post-build script, srec_cat application, and bootloader binary, so the IAR build with the current configuration succeeds.